extensions/toolbox/src/index.tsx

```diff
@@ -0,0 +1,208 @@
+
+  useEffect(() => {
+    const categorys: Category[] = [];
+    Object.entries(scripts).forEach(([categoryName, categoryValue]) => {
+      const category: Category = {
+        title: categoryName,
+        items: [],
+      };
+      categorys.push(category);
+      Object.entries(categoryValue).forEach(([, scriptItem]) => {
+        category.items.push(scriptItem);
+      });
+    });
+    setCategorys(categorys);
+  }, []);
+
+  return (
+    <List isLoading={categorys === undefined}>
+      {categorys?.map((category) => (
+        <List.Section key={category.title} title={category.title}>
+          {category.items.map((item) => {
+            return <ListItem item={item} key={item.info.title} />;
+          })}
+        </List.Section>
+      ))}
+    </List>
+  );
+}
+
+const ListItem = React.memo(function ListItem(props: { item: Script }) {
+  const { push } = useNavigation();
+
+  const item = props.item;
+  const info = item.info;
+
+  const keyword = useMemo(
+    () =>
+      info.keywords?.reduce((keywordArray, currentKeyword) => {
+        currentKeyword.split("")?.reduce((preString, currentString) => {
+          keywordArray.push(preString + currentString);
+          return preString + currentString;
+        }, "");
+        return keywordArray;
+      }, [] as string[]),
+    [info.keywords]
+  );
+
+  async function action(config: { useClipboard: boolean; isWindowClose: boolean }) {
+    selectScript = item.run;
+    if (config.useClipboard) {
+      const content = await isClipboardContent();
+      if (content) {
+        const result = await runScript({ content: content, isWindowClose: config.isWindowClose });
+        if (!result) {
+          push(<InputView info={info} />);
+        } else {
+          selectScript = null;
+        }
+      } else {
+        push(<InputView info={info} />);
+      }
+    } else {
+      push(<InputView info={info} />);
+    }
+  }
+
+  return (
+    <List.Item
+      title={info.title}
+      accessoryTitle={info.desc}
+      keywords={keyword}
+      icon={info.icon}
+      actions={
+        <ActionPanel>
+          {(info.type === "all" || info.type === "clipboard") && (
+            <ActionPanel.Item
+              title={"Run Script Clipboard"}
+              icon={Icon.Clipboard}
+              onAction={async () => {
+                action({ useClipboard: true, isWindowClose: true });
+              }}
+            />
+          )}
+          {(info.type === "all" || info.type === "input") && (
+            <ActionPanel.Item
+              title={"Run Script Input"}
+              icon={Icon.Text}
+              onAction={async () => {
+                action({ useClipboard: false, isWindowClose: false });
+              }}
+            />
+          )}
+          {(info.type === "all" || info.type === "clipboard") && (
+            <ActionPanel.Item
+              shortcut={{ modifiers: ["cmd"], key: "s" }}
+              title={"Run Script - Window Keep"}
```

Comment:
A better title would be `Run Script - Keep Window`

Reply:
Thank you. I fixed it.